docs/accessibility/content.md

```diff
@@ -157,9 +157,42 @@ Microcopy comprises short words, phrases, or chunks of text used on the web or i
 
 Best practices for all microcopy:
 
-- Keep text short but descriptive. Use as few words as possible, but avoid non-informative or vague text, like “click here,” “learn more,” “read more,” etc.
-- Make the first word meaningful. For example, avoid starting with “a”, “the”, “for”, etc. In many cases, this practice allows screen reader users to navigate items alphabetically.
-- Keep the text consistent. Ensure that consistency is maintained on pages, sites, and entire user journeys.
+- Keep text short but descriptive
+- Use as few words as possible
+- Avoid non-informative or vague text, like “click here,” “learn more,” “read more,” etc.
+- Make the first word meaningful
+    - For example, avoid starting with “a”, “the”, “for”, etc
+    - In many cases, this practice allows screen reader users to navigate items alphabetically
+- Keep the text consistent.
+    - Ensure that consistency is maintained on pages, sites, and entire user journeys.
+
+### Creating emphasis
+
+When writing for the web, use bold and italic formatting sparingly. Remember: if everything is emphasized, nothing is emphasized.
+
+Red Hat design guidelines mandate users employ either bold or italics—never both together.
+
+#### Bolding
+
+The most effective way to highlight key points in text is by **using bold**. Bold text naturally draws the eye and makes specific parts stand out.
+
+However, the golden rule for bolding is moderation. Overusing bold can backfire—if everything is bold, nothing feels important. Instead, choose a few key words or a single, short sentence that encapsulates your main message.
+
+#### Italics
+
+Italics offer another useful option for formatting, though they’re not a substitute for bold. They can help distinguish a section of text from the rest, adding emphasis in a subtler way. 
+
+However, italics can be harder to read, especially for individuals with dyslexia. For this reason, and like bolding, it’s best to use italics sparingly.
+
+#### Underlines
+
+Do not underline any text that isn’t a link. Underlining text that is not a link is confusing to users and must be avoided. Instead of underlining text, consider bolding or italicizing it.
+
+Also, be cautious about using your site’s link color elsewhere—especially for text. Users may think blue text is a link even if it is not. Catch up on how to use [color accessibly](/accessibility/design/#color) on our [Design](/accessibility/design/) page.
+
+#### All caps
+
+Only use all caps for acronyms and abbreviations (like <abbr title="Red Hat Design System">RHDS</abbr> and <abbr title="Consumer Financial Protection Bureau">CFPB</abbr>). Do not use all caps for emphasis—use bold or italics instead. For users with dyslexia, the changes in letter heights from big to small is actually easier to read—especially when the copy is longer than 2-3 words.
```

Comment:
`emphasis—use` → I'd probably replace the em dash with a semicolon or period. 

`changes in letter heights from big to small is actually` → `variation in letter height between uppercase and lowercase is actually`

`2-3` → `two or three`